### ISSUE TITLE
Plan

### ISSUE THREAD

**Issue by sarahaguasvivas** [sarahaguasvivas/open_ended_learning#1]
GIVEN: 
-  Neural network (letnet structure maybe?) trained to be an expert of only M classes
-  Dataset with N classes

          For each step:
                    INPUT: Single data sample, current belief network with the M classes

                    PROCESS:
                                  - Determine if label has been seen before 
                                  if True:
                                           Backprop using (*)
                                  else:
                                           Add new class to Bayesian Belief Network
                                           Perform (**)

                    OUTPUT: 
                                     neural network, new belief network

Unanswered questions
(*) How to map actual nn labels with new classes
(**) Build the new weights for the Bayesian Belief Network
(***) Neural Network?
